Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -28,6 +28,14 @@ struct K {
     struct vcId {
         static let vcIdWebViewLogin = "WebViewLoginViewController"
     }
+    
+    struct constant {
```

Comment:
I think that this is not a good place ( inside `UniversalLoginViewController`). We have `UtilsUrls`and I think we should make all work related with URLs there.

Reply:
Ok, we will consider it in next version